v2.0/deploy-a-test-cluster.md

```diff
@@ -112,60 +114,7 @@ Before getting started, it's important to review some limitations and requiremen
 
 {{site.data.alerts.callout_success}}With the <code>cockroach</code> binary on your local machine, other client <a href="cockroach-commands.html"><code>cockroach</code> commands</a> can be run in the same way.{{site.data.alerts.end}}
 
-## Step 3. Start a load generator
-
-CockroachDB provides a number of [example load generators](https://github.com/cockroachdb/loadgen) in Go for simulating client workloads. The program you'll use here is called [`rand`](https://github.com/cockroachdb/loadgen/tree/master/rand). It will generate and insert random data into the `bank.accounts` table you created in the previous step.
-
-1. In a new terminal, install [Go](https://golang.org/dl/). If you're on a Mac and using Homebrew, use `brew install go`.
-
-2. Install the `rand` load generator:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ go get github.com/cockroachdb/loadgen/rand
-    ~~~
-
-3. Start the `rand` load generator:
-
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ rand --host=Cockroach-ApiLoadB-LVZZ3VVHMIDA-1266691548.us-west-2.elb.amazonaws.com bank accounts
-    ~~~
-    - Set the `--host` flag to just the hostname portion of the **ConnnectionString** from the CloudFormation UI.
-    - Be sure to list `bank accounts` **after** the `--host` flag.
-
-4. Back in the terminal with the SQL shell, check the random data that `rand` is inserting into the `bank.accounts` table:
-
-    ~~~ sql
-    > SELECT * FROM bank.accounts LIMIT 10;
-    ~~~
-
-    ~~~
-    +-------+---------+
-    |  id   | balance |
-    +-------+---------+
-    |     1 | 1000.50 |
-    |  5250 |      84 |
-    |  6205 |      60 |
-    | 13086 |       4 |
-    | 16745 |      12 |
-    | 19124 |      85 |
-    | 25573 |      54 |
-    | 26957 |      46 |
-    | 31117 |      65 |
-    | 37195 |      59 |
-    +-------+---------+
-    (10 rows)
-    ~~~
-
-5. Exit the SQL shell:
-
-    {% include copy-clipboard.html %}
-    ~~~ sql
-    > \q
-    ~~~
-
-## Step 4. Monitor the cluster
+## Step 3. Monitor the cluster
 
 You can use the cluster's [Admin UI](admin-ui-overview.html) to monitor the `rand` workload and overall cluster behavior.
```

Comment:
nit: We don't know which load gen they chose, so we should probably just remove `rand`.

Reply:
Done.